src/table/utils/__tests__/handle-key-press.spec.ts

```diff
@@ -623,10 +627,16 @@ describe('handle-key-press', () => {
       totalsPosition = 'noTotals';
       jest.spyOn(handleAccessibility, 'focusSelectionToolbar').mockImplementation(() => jest.fn());
       jest.spyOn(handleAccessibility, 'announceSelectionState').mockImplementation(() => jest.fn());
+      jest.spyOn(handleAccessibility, 'copyCellValue');
       jest.spyOn(handleScroll, 'handleNavigateTop').mockImplementation(() => jest.fn());
     });
 
-    afterEach(() => jest.clearAllMocks());
+    afterEach(() => {
+      jest.clearAllMocks();
+      flags = {
+        isEnabled: () => true,
+      };
```

Comment:
doesn't really make sense to do this in the afterEach, should be in the beforeEach